### ISSUE TITLE
Don't export run

### ISSUE THREAD

**Issue by mortenpi** [soumitradev/TermLayouts.jl#2]
It conflicts with the export from `Base`, so exporting it here probably doesn't serve much purpose?

```
julia> using TermLayouts; run()
WARNING: both TermLayouts and Base export "run"; uses of it in module Main must be qualified
ERROR: UndefVarError: run not defined
```

And the docs anyway recommend running with `TermLayouts.run()`, right?

**Comment by soumitradev** [comment]
The docs do recommend running with `TermLayouts.run()` but I think it would still be better to steer away from any name conflicts. We discussed using `activate()` first, but I think @johnnychen94 advised against that. Any ideas for alternatives?

**Comment by mortenpi** [comment]
I don't think the conflict with `run` is a user-facing problem -- it only affects this code base (so you'd have to do `Base.run` here). Just unexporting the function would be enough to solve the user-facing problem.

**Comment by soumitradev** [comment]
Fixed in a0d9aad
